pkg/flagbuilder/build_flags.go

```diff
@@ -131,6 +131,12 @@ func BuildFlags(options interface{}) (string, error) {
 				flag = fmt.Sprintf("--%s=%s", flagName, vString)
 			}
 
+		case *string:
+			if v != nil {
+				vString := fmt.Sprintf("%v", *v)
```

Comment:
So actually I think _this_ is fine; we are adding a case here, and we return error on unknown cases, so we know this is additive.   I would prefer us to honor flagEmpty, but we can add that when we first need it.